README.md

```diff
@@ -32,6 +32,11 @@ To install dependencies, run this at the repository root:
 yarn run bootstrap
 ```
 
+To build the project, run this at the repository root:
```

Comment:
I would include something about this being a required step for many (all?) commands

Reply:
Good point, the reader may decide to be a smartass about it